variables.tf

```
@@ -215,3 +215,9 @@ variable "vm_tags" {
   type        = map(string)
   default     = {}
 }
+
+variable "availability_set_id" {
+  description = "Specifies the ID of the Availability Set in which this Virtual Machine should exist."
```

Nitpicky, but could update description to be even more similar to existing variables:

```suggestion
  description = "The ID of an availability set in which this VM should be created."
```

---

Done